rfcs/2021-19-02-units-for-typography-and-layout.md

```diff
@@ -0,0 +1,603 @@
+| ------------------------------------------------ | :-------------------: | :------------------: | :---------------: | :--------------: |
+| Everything px(web)                               |           ❌           |          ❌           |         ❌         |        ✅         |
+| Everything rems(web)                             |           ✅           |          ✅           |         ⚠️         |        ✅         |
+| Tyography rems, layout px(web)                   |           ✅           |          ❌           |         ✅         |        ✅         |
+| Everything dp(native)                            |           ❌           |          ✅           |         ❌         |        🚫         |
+| Tyography auto scale, layout pixel ratio(native) |           ✅           |          ❌           |         ✅         |        🚫         |
+
+>⚠️ Doesn't breaks but doesn't renders as expected
+### Accessibility Guideline?
+The WCAG guideline says that content satisfies the **success criterion** if it can be **scaled up to 200%**. It can go beyond that but logically beyond that the scaling becomes more extreme and adaptive layouts may introduce usability problems.
+
+For example, words may be too wide to fit into the horizontal space available to them, causing them to be truncated; layout constraints may cause text to overlap with other content when it is scaled larger; or only one word of a sentence may fit on each line, causing the sentence to be displayed as a vertical column of text that is difficult to read.
+>🔗 [WCAG guideline link](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html)
+### What units other Design Systems are using?
+
+| Name                                                                                     | Typography | Layout | Comments                                 |
+| ---------------------------------------------------------------------------------------- | ---------- | ------ | ---------------------------------------- |
+| [Primer - GitHub](https://primer.style/css/support/spacing)                              | px         | px     | NA                                       |
+| [Spectrum - Adobe](https://react-spectrum.adobe.com/react-spectrum/getting-started.html) | px         | px     | revisitng decision                       |
+| [Braid - Seek](https://seek-oss.github.io/braid-design-system/)                          | px         | px     | revisiting decision                      |
+| [Styled System](https://styled-system.com/)                                              | px         | px     | default px, can change by suffixing unit |
+| [Facebook](https://twitter.com/naman34/status/1362626358539481090)                       | rems       | px     | NA                                       |
+| [Chakra](https://chakra-ui.com/docs/getting-started)                                     | rems       | rems   | default rems, option for other units     |
+
+>📝 Note: Couldn't find anything similar for react native
```

Comment:
In case you want to explore:

- [Wix](https://github.com/wix/react-native-ui-lib)
- [Ant](https://github.com/ant-design/ant-design-mobile-rn)
- [Fluent](https://github.com/microsoft/fluentui-react-native)